src/fromager/settings.py

```diff
@@ -96,6 +96,11 @@ def build_dir(self, pkg: str, sdist_root_dir: pathlib.Path) -> pathlib.Path:
             return sdist_root_dir / relative_build_dir
         return sdist_root_dir
 
+    def build_tag(self, pkg: str, version: Version | str) -> int:
```

Comment:
I think we said we wanted the variant to be part of this query, too.

Reply:
Oh i thought we were doing the per variant settings after pydantic lands in so that we could port all the fields

Reply:
OK, we can do that. I didn't have that sequencing in mind.